tools/preflight/preflight.go

```diff
@@ -774,6 +778,11 @@ func (o *Run) createSnapshotFromPVC(ctx context.Context, volSnapName, volSnapCla
 	pvcName, uid string, clients ServerClients) (*unstructured.Unstructured, error) {
 	volSnap := createVolumeSnapsotSpec(volSnapName, volSnapClass, o.Namespace, snapshotVer, pvcName, uid)
 	if err := clients.RuntimeClient.Create(ctx, volSnap); err != nil {
+		volSnapYAML, yErr := toYAML(volSnap)
+		if yErr != nil {
+			return nil, yErr
+		}
+		o.Logger.Warnf("volume snapshot YAML: \n%s", string(volSnapYAML))
```

Comment:
Here also the creation failure log stmt should be consistent with other resource types logs.